broker/src/session/session.erl

```diff
@@ -496,7 +496,7 @@ finalize({failed, Reason}, State = #state{session = Session = #session{call_log
     {internal_error, Details} -> [{fail_reason, "internal error"}, {fail_details, io_lib:format("~p", [Details])}];
     _ ->                         [{fail_reason, "unknown error"}]
   end,
-  CallLog:update([{state, NewState}, {finished_at, calendar:universal_time()}] ++ FailInfo),
+  ok = CallLog:update([{state, NewState}, {finished_at, calendar:universal_time()}] ++ FailInfo),
```

Comment:
This change doesn't hurt, but it doesn't ensure the update went well, because the `ok` comes from [here](https://github.com/instedd/verboice/blob/77fa26385a72e82dd3f67f259db712c5632bdc62/broker/src/call_log_srv.erl#L31).

> [gen_server:cast/2](https://www.erlang.org/doc/man/gen_server.html#cast-2): Sends an asynchronous request to the ServerRef of the gen_server process and returns ok immediately, ignoring if the destination node or gen_server process does not exist.